Change the default polling trigger to 1 second (#8751)
spring-projects/spring-integration: src/reference/antora/modules/ROOT/pages/whats-new.adoc

- The `LockRegistry` provides template-like API to execute provided task while locked.
See xref:distributed-locks.adoc[Distributed Locks] for more information.

> - The default trigger for polling endpoint is now a `PeriodicTrigger` instance of 1-second fixed delay period instead of very tight 10 milliseconds before.

Reviewer: ```suggestion
- The default trigger for polling endpoint is now a `PeriodicTrigger` instance with a 1 second fixed delay period; previously, the default was 10 milliseconds.
```
